docs/environments/unarchived-environment.md

```diff
@@ -0,0 +1,17 @@
+---
+draft: true
+title: Unarchive an Environment
```

Comment:
Please keep naming convention on the file name. It's named as `unarchiveD` instead of an infinitive verb, as most of the files in our docs code base.

Reply:
@jfoliveira file name updated